### PR TITLE
Add Harbor v2.8.2 and Contour v1.24.4

### PR DESCRIPTION
**Pull request by sally5c**
Add links to download Harbor v2.8.2 and Contour v1.24.4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,6 @@ Prior vSphere 8 Update 1, the Supervisor Services are only available with Superv
 | Kubernetes Ingress Controller Service - Contour | ❌ | ✅ |
 | External DNS Service - External DNS | ❌ | ✅ |
 *\* The embedded Harbor Registry feature is still available and supported on vSphere 7 and onwards.*
-    
-## vSAN Data Persistence Platform (vDPP):
```
Comment (YuxinZhou) [inline README.md:33]: Thank you for catching the duplicate doc.

**Review by YuxinZhou — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -118,11 +116,13 @@ Contour is an Ingress controller for Kubernetes that works by deploying the Envo
...
 ### Contour Versions
 
-- Download latest version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour.yml)
+- Download latest version: [Contour v1.24.4](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.24.4/contour.yml)
+- Download version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour.yml)
 
 Contour Sample `values.yaml`
 
-- Download the latest version: [values for v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour-data-values.yml). These values can be used _as-is_ and require no configuration changes.
+- Download latest version: [values for v1.24.4](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.24.4/contour-data-values.yml). These values can be used _as-is_ and require no configuration changes.
```
Comment (douglas-c) [inline README.md:124]: Are the default data-values files identical for both versions? If so maybe we can skip versioning these for now and just link one.

**Review by douglas-c — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sally5c) [reply README.md:124]: Yes, they are the same data values. Do you also want to move contour/v1.18.2/contour-data-values.yml up one level to contour/contour-data-values.yml path?

Comment (douglas-c) [reply README.md:124]: Are you referring to the path on jfrog? It might not matter from this repo, we could just point at the latest.

Comment (sally5c) [reply README.md]: Yeah, I meant the path on jfrog.

**Pull request merged**